app/src/androidTest/java/org/mozilla/fenix/ui/ContextMenusTest.kt

```diff
@@ -128,6 +128,30 @@ class ContextMenusTest {
         }
     }
 
+    @Ignore("Test failures: https://github.com/mozilla-mobile/fenix/issues/12473")
```

Comment:
@AndiAJ is this still needed since https://github.com/mozilla-mobile/fenix/issues/18421 has been fixed?

Reply:
Hi @boek , sorry for the late reply. (I was in PTO)

Yes, it can be removed.

The test will need some minor refactoring work so I'll open a PR and run it a couple of times on Firebase to be sure everything work properly.